ada_feeding/README.md

```diff
@@ -7,55 +7,58 @@ The `ada_feeding` package contains the overarching code to run the robot-assiste
-    2. Real nodes: Follow the instructions in the [`ada_feeding_perception` README](https://github.com/personalrobotics/ada_feeding/blob/ros2-devel/ada_feeding_perception/README.md#usage)
-5. Run the action servers:
-    1. If the e-stop is plugged in: `ros2 launch ada_feeding ada_feeding_launch.xml`
-    2. If the e-stop is not plugged in: `ros2 launch ada_feeding ada_feeding_launch.xml use_estop:=false`
-6. Launch MoveIt2:
-    1. Sim (RVIZ): `ros2 launch ada_moveit demo_feeding.launch.py sim:=mock`
-    2. Real Robot: `ros2 launch ada_moveit demo_feeding.launch.py`
-7. Test it:
-    1. Test the individual actions with the command line interface:
-        1. `ros2 action send_goal /MoveAbovePlate ada_feeding_msgs/action/MoveTo "{}" --feedback`
-        2. `ros2 action send_goal /AcquireFood ada_feeding_msgs/action/AcquireFood "{header: {stamp: {sec: 0, nanosec: 0}, frame_id: ''}, detected_food: {roi: {x_offset: 0, y_offset: 0, height: 0, width: 0, do_rectify: false}, mask: {header: {stamp: {sec: 0, nanosec: 0}, frame_id: ''}, format: '', data: []}, item_id: '', confidence: 0.0}}" --feedback`
-        3. `ros2 action send_goal /MoveToRestingPosition ada_feeding_msgs/action/MoveTo "{}" --feedback`
-        4. `ros2 action send_goal /MoveToStagingConfiguration ada_feeding_msgs/action/MoveTo "{}" --feedback`
-        5. `ros2 action send_goal /MoveToMouth ada_feeding_msgs/action/MoveToMouth "{}" --feedback`
-        6. `ros2 action send_goal /MoveFromMouth ada_feeding_msgs/action/MoveTo "{}" --feedback`
-        7. `ros2 action send_goal /MoveToStowLocation ada_feeding_msgs/action/MoveTo "{}" --feedback`
-    2. Test the individual actions with the web app:
-        1. Launch the web app ([instructions here](https://github.com/personalrobotics/feeding_web_interface/tree/main/feedingwebapp))
-        2. Go through the web app, ensure the expected actions happen on the robot.
-    3. Test the watchdog in isolation:
-        1. Echo the watchdog topic: `ros2 topic echo /ada_watchdog`
-        2. Induce errors in the force-torque sensor and verify the watchdog reacts appropiately. Errors could include:
-            - Terminating the node.
-            - Disconnecting the physical force-torque sensor from power.
-            - Inducing corruption by causing the dummy note to output zero-variance values: `ros2 param set /dummy_ft_sensor std  '[0.1, 0.1, 0.0, 0.1, 0.1, 0.1]'`
-    4. Test the watchdog with the action servers:
-        1. Start an action (see above), induce errors in the force-torque sensor (see above), and ensure the action gets aborted.
-        2. While there are errors in the force-torque sensor, ensure no new goals get accepted.
-        3. Terminate the watchdog node and ensure in-progress actions get aborted and incoming goals get rejected.
-        4. Launch the action servers without the watchdog node running and ensure it rejects all goals.
+1. Source your workspace: `source install/setup.bash`
```

Comment:
Looks like something went wrong with the numbering here.

Reply:
Thanks for the heads-up! I had to add the `--number` flag to `mdformat`. That addressed this and the below comment.